pkgs/by-name/st/streamcontroller/package.nix

```diff
@@ -54,7 +62,7 @@ stdenv.mkDerivation rec {
     mkdir -p "$out/etc/udev/rules.d"
     cp ./udev.rules $out/etc/udev/rules.d/70-streamcontroller.rules
 
-    install -D ./flatpak/icon_256.png $out/share/icons/hicolor/256x256/apps/streamcontroller.png
+    install -D ./flatpak/icon_256.png $out/share/icons/hicolor/256x256/apps/com.core447.StreamController.png
```

Comment:
This name is picked up for the trayicon as well so I'm just following the upstream name